templates/calculator/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Calculator</title>
```

Comment:
please translate, our customers speak only German, the required text is "Taschenrechner"